contrib/experimental/great_expectations_experimental/expectations/expect_column_value_to_meet_frequency.py

```diff
@@ -0,0 +1,129 @@
+"""
+This is a template for creating custom ColumnAggregateExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations
+"""
+
+from typing import Dict, Optional
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.exceptions import InvalidExpectationConfigurationError
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnAggregateExpectation
+from great_expectations.expectations.metrics import (
+    ColumnAggregateMetricProvider,
+    column_aggregate_partial,
+    column_aggregate_value,
+)
+
+
+# This class defines a Metric to support your Expectation.
+# For most ColumnAggregateExpectations, the main business logic for calculation will live in this class.
+class ColumnValueMeetsFrequency(ColumnAggregateMetricProvider):
+    # This is the id string that will be used to reference your Metric.
+    metric_name = "column.value_meets_frequency"
+    # Values passed in to our expectation used in our metric
+    value_keys = ('value',)
+
+    # This method implements the core logic for the PandasExecutionEngine
+    @column_aggregate_value(engine=PandasExecutionEngine)
+    def _pandas(cls, column, value, **kwargs):
+        
+        return column.value_counts()[value] / len(column)
+    
+
+# This class defines the Expectation itself
+class ExpectColumnValueToMeetFrequency(ColumnAggregateExpectation):
```

Comment:
nit (non-blocking): May I suggest a more verbose and self-descriptive name for this?
```suggestion
class ExpectColumnOccuranceOfValueToMeetFrequency(ColumnAggregateExpectation):
```